### PR TITLE
Update changelog for 0.9.0

### PR DESCRIPTION
**Pull request by nmoroze**

### CODE REVIEW

File: Changes [hunk 1]
```diff
@@ -8,6 +8,49 @@ The changes in each SiliconCompiler release version are described below. Commit
...
+**Minor:**
+
+* Schema: Added ability to store per-parameter designer notes.
+* Added offline wheels distribution.
+* Added checklist support.
+* Added ``read_lef()`` function to help with PDK bring-up.
+* Added environment variables to replay scripts.
+* Added LVS/DRC signoff flow and top-level GDS stream out flow.
+* Added native support for Sky130 I/O library, along with Heartbeat + padring example.
+* Added automatic capture of peak memory usage.
```
Comment (WRansohoff) [inline Changes:46]: Might be worth mentioning that this introduced a new dependency, [psutil](https://psutil.readthedocs.io/en/latest/).

**Review by WRansohoff — APPROVED** [review]
Looks good to me, thanks for writing this up!

File: Changes (hunk 1 above)
Comment (nmoroze) [reply Changes]: Good idea, thanks! Added note and moved this bullet to "Major" heading for visibility.

**Review by aolofsson — APPROVED** [review]
Looks good

**Pull request merged**